src/Packages/Packages.csproj

```diff
@@ -30,6 +33,9 @@
     <NuGetSpec Include="..\WebJobs.Extensions.SendGrid.NuGet\WebJobs.Extensions.SendGrid.nuspec">
       <Link>WebJobs.Extensions.SendGrid\WebJobs.Extensions.SendGrid.nuspec</Link>
     </NuGetSpec>
+    <NuGetSpec Include="..\WebJobs.Extensions.Twilio.NuGet\WebJobs.Extensions.Twilio.nuspec">
+      <Link>WebJobs.Extensions.Twilio\WebJobs.Extensions.Twilio.nuspec</Link>
+    </NuGetSpec>
```

Comment:
How were these getting built? Was the official build doing something different?

Reply:
Twilio was just missing from sln. Build proj files still included it. So this is just to make it easier to see/modify the files.